source/logs/command-history.txt

```diff
@@ -32,3 +32,36 @@ text editor:
 
    * - Windows
      - ``%UserProfile%/.mongodb/mongosh/mongosh_repl_history``
+
+Starting in |mdb-shell| 2.4.0, you can use the ``history()`` command to
+view a list of previously executed commands. For example:
+
+.. code-block:: javascript
+
+   history()
+
+Example output, which shows a list of commands in an array of strings:
```

Comment:
[nit] Can we make this into a complete sentence? 
```suggestion
The following example output shows a list of commands in an array of strings:
```

Reply:
Done